docs/articles/expensify-classic/bank-accounts-and-payments/payments/Receive-and-Pay-Bills.md

```diff
@@ -16,6 +16,8 @@ Share your Expensify billing email with vendors to receive bills automatically.
 - Ask vendors to email bills to your billing address: `domainname@expensify.cash` (e.g., for *expensify.com*, use `expensify@expensify.cash`).  
 - Once emailed, the bill is automatically created in Expensify, ready for payment.  
 
+![Setting the Primary Contact at Domain Admins > Primary Contact](https://help.expensify.com/assets/images/OldDot%20-%20Create%20%26%20Pay%20Bills%201.png){:width="100%"}
```

Comment:
NAB: It's a bit odd how OldDot says `The Primary Contact is the main point of contact for expensicorp.com and will receive bills sent to this domain's bill pay address expensicorp.com@expensify.cash`, but the format given by the example above suggests the address would be `expensicorp@expensify.cash`.